index.html

```diff
@@ -2,15 +2,15 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <title>Fork Me! FCC: Test Suite Template</title>
+    <title>Review Project</title>
     <link rel="stylesheet" href="styles.css" />
   </head>
 
   <body>
     <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
     <div class="container">
       <header class="header">
-        <h1 id="title" class="text-center">Francis and Andile form</h1>
+        <h1 id="title" class="text-center">Francis Survey form</h1>
         <p id="description" class="description text-center">
           Thank you for taking the time to help us improve the platform
         </p>
```

Comment:
Please always try to limit using long words for classes and IDs, so as not to run into problems.😊

Reply:
- [OPTIONAL] This comment is marked optional but highly recommended to heed the instructions here for the benefit of future review. The changes you make reflect on the right side where I wrote _*Review should be done here.*_. That's where you have green highly not other way round. Please, take this instructions very seriously so that you would not make the same mistake when next you review someone's code.

![review-side](https://user-images.githubusercontent.com/26597432/193638023-0712784f-e5c7-493e-b985-a1ad3599d91f.png)